hack/add_addresses.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+while true; do
```

Comment:
what does this script do? It might be worth adding some comments

Reply:
Done.